src/BasisTheory.ts

```diff
@@ -328,6 +328,10 @@ export class BasisTheory
     return assertInit(this._logs);
   }
 
+  /**
+   * @deprecated Reactor Formulas are now deprecated and will be removed in a future release.
+   * @docs For more details visit [our API reference](https://developers.basistheory.com/docs/api/reactors#create-reactor).
```

Comment:
Can we have a line here (and elsewhere) to say people should use the `code` property directly?